float64 goal_syn
float64 goal_duration
---
bool answer